src/EditorFeatures/Core/ReferenceHighlighting/ReferenceHighlightingViewTaggerProvider.cs

```diff
@@ -163,7 +163,8 @@ protected override Task ProduceTagsAsync(TaggerContext<NavigableHighlightTag> co
             var document = documentHighlights.Document;
 
             var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
looks like we can remove this line entirely then, right?

Reply:
Ah yes, we can. Also made the containing method sync again.